Example template not working

Hey, 

Tested https://deno.land/x/abc/examples/02_template/main.ts

With the Deno version I am using 

--
# deno --version
deno 0.30.0
v8 8.1.108
typescript 3.7.2
--

It fails with:

deno_1  | error TS2322: Type 'void' is not assignable to type 'string'.
deno_1  | 
deno_1  | ► file:///opt/app/http-abc.ts:15:5
deno_1  | 
deno_1  | 15     return await c.render("./index.html", { name: "zhmushan" })

--- a/examples/01_cat_app/cat_handler.ts
+++ b/examples/01_cat_app/cat_handler.ts
@@ -10,7 +10,9 @@ export function findAll(): Cat[] {
 
 export function findOne(c: Context): Cat {
   const { id } = c.params as { id: string };
-  return cats.find((cat: Cat): string => cat.id.toString() === id);
+  return cats.find((cat: Cat): void => {
+    cat.id.toString() === id;
+  });
 }
 
 export async function create(c: Context): Promise<Cat> {

--- a/examples/01_cat_app/cat_handler.ts
+++ b/examples/01_cat_app/cat_handler.ts
@@ -10,8 +10,8 @@ export function findAll(): Cat[] {
 
 export function findOne(c: Context): Cat {
   const { id } = c.params as { id: string };
-  return cats.find((cat: Cat): void => {
-    cat.id.toString() === id;
+  return cats.find((cat: Cat): boolean => {
+    return cat.id.toString() === id;
   });
 }
 

--- a/examples/02_template/main.ts
+++ b/examples/02_template/main.ts
@@ -10,9 +10,12 @@ app.renderer = {
 };
 
 app
-  .get("/", (c: Context): Promise<void> => {
-    return c.render("./index.html", { name: "zhmushan" });
-  })
+  .get(
+    "/",
+    (c: Context): Promise<void> => {
+      return c.render("./index.html", { name: "zhmushan" });
+    }
+  )
   .start({ port: 8080 });
 
 console.log(`server listening on 8080`);

--- a/examples/02_template/main.ts
+++ b/examples/02_template/main.ts
@@ -10,9 +10,9 @@ app.renderer = {
 };
 
 app
-  .get("/", (c: Context): string =>
-    c.render("./index.html", { name: "zhmushan" })
-  )
+  .get("/", (c: Context): Promise<void> => {
+    return c.render("./index.html", { name: "zhmushan" });
+  })
   .start({ port: 8080 });
 
 console.log(`server listening on 8080`);

